tests/util/pragma.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const assert = require('assert');
+const SourceCode = require('eslint').SourceCode;
+const espree = require('espree');
```

Comment:
is there any way to do this with eslint itself, rather than adding a dev dep on espree?

Reply:
Not as far as I could tell, unfortunately. I tried using eslint first, but eslint does not expose the API to generate the AST object needed to create the SourceCode object.